python/datashaper/datashaper/engine/verbs/unhot.py

```diff
@@ -29,11 +29,11 @@ def unhot(
 
     for col in output.columns:
         try:
-            indexValue = columns.index(col)
+            columns.index(col)
         except ValueError:
             filteredList.append(col)
 
-    if preserveSource == False:
+    if not preserveSource:
```

Comment:
now that you are renaming maybe rename this to `preserve_source`

Reply:
preserveSource is a little tricky because the JSONSchema uses camelcase. We'll need to convert from camel->snake as we read. It would be more idiomatic, but I don't think it's high priority.